akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/indices/ibex35/Ibex35HistoricManager.scala

```diff
@@ -52,10 +53,10 @@ class Ibex35HistoricManager extends Retriever with Stash {
       )
 
     case DataRetrieved(index, typeName, data, _) =>
-      if(data.isEmpty) {
+      data.hits.headOption.fold {
         log.info("Starting without initial date")
         self ! ReadyToStart(None)
-      } else {
+      } { head =>
```

Comment:
If head it's not used, just leave an underscore

Reply:
Done